docs/angular/fundamentals/develop-like-google.md

```diff
@@ -341,6 +342,18 @@ The `tags` array is used to impose constraints on the dependency graph. Read mor
 
 Nx uses its advanced code analysis to construct a dependency graph of all applications and libraries. Some dependencies, however, cannot be determined statically. You can use the `implicitDependencies` array to list the dependencies that cannot be determined statically.
 
+The `ignore` property is _optional_, but can defined to ignore additional files and folders during the `affected:*` commands.
```

Comment:
you should update the react and web guides as well

Reply:
Will do